src/open_inwoner/react/modules/SideNavModule/SideNavModule.tsx

```diff
@@ -0,0 +1,41 @@
+import { Root } from 'react-dom/client'
+import SideNav from '../../components/SideNav/SideNav'
```

Comment:
Please use the aliases we configured 😄 
```suggestion
import SideNav from '@react/components/SideNav/SideNav'
```